en/appendices/5-0-migration-guide.rst

```diff
@@ -312,6 +313,23 @@ The finder can then be called as ``find('byCategoryOrUser', userId: $somevar)``.
 You can even include the special named arguments for setting query clauses.
```

Comment:
```suggestion
You can even include custom named arguments for setting query clauses.
```
I guess, since we are at it.

Reply:
Can also be a different PR.

Reply:
You know you are free to create your own PR's on this repo as well, right? 😛